generic-sensor/resources/generic-sensor-helpers.js

```diff
@@ -92,6 +93,11 @@ function verifyGeoSensorReading(pattern, {latitude, longitude, altitude,
     accuracy, altitudeAccuracy, heading, speed], timestamp, isNull);
 }
 
+function verifyProSensorReading(pattern, {distance, max, near, timestamp},
```

Comment:
Abbreviating "proximity" to "pro" is confusing. I'd rather call this `verifyProximitySensorReading()`.

Reply:
Done.